tests/features/common/clustering.feature

```diff
@@ -1,4 +1,4 @@
-@rhpam-7/rhpam71-businesscentral-openshift @rhpam-7/rhpam71-businesscentral-monitoring-openshift @rhdm-7/rhdm71-decisioncentral-openshift
+@rhpam-7/rhpam72-businesscentral-openshift @rhpam-7/rhpam72-businesscentral-monitoring-openshift @rhdm-7/rhdm72-decisioncentral-openshift
```

Comment:
Instead of replacing rhpam71 with rhpam72 images to be tested, shouldn't both be present? (So that both rhpam71 and rhpam72 images be included in the test suite?)

Reply:
Can be, but for that we can use the 7.1.x branch.
And 7.1 will be deprecated a little while after the 7.2 is released.
and IMHO, we will not build 7.1 images with master branch.

Reply:
Ah yes, good point. Thanks!